modules/vertx-swagger-router/src/main/java/io/vertx/ext/swagger/router/SwaggerRouter.java

```diff
@@ -29,6 +29,7 @@
 
     private static Logger VERTX_LOGGER = LoggerFactory.getLogger(SwaggerRouter.class);
 
+    private static Pattern PATH_PARAMETER_NAME = Pattern.compile("\\{(\\p{Alpha}[\\p{Alpha}\\d]*)\\}");
```

Comment:
Indeed! I had an impression that it doesn't allow parameters which start with number.
But I will double check again just in case.